### PR TITLE
LPS-186032 Create a SF Check for Replace with DLFileEntryLocalService.getFileAsStream()

### PR DESCRIPTION
**Pull request by TamyrisTorres**
LPS-186032
https://issues.liferay.com/browse/LPS-186032

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by TamyrisTorres** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFileAsStreamCheck.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaFileAsStreamCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if ((content == null) || !fileName.endsWith(".java")) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFileAsStreamCheck.java:38]: @TamyrisTorres, please see https://github.com/natocesarrego/liferay-portal/pull/1255/commits/ee5454db602684e8d14e7ffc437477c9c8499afe#r1213573113

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFileAsStreamCheck.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+		String newContent = content;
+
+		Matcher getFileMatcher = _getFilePattern.matcher(content);
+
+		while (getFileMatcher.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, getFileMatcher.start());
+
+			if (_validateClassName(
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFileAsStreamCheck.java:50]: @TamyrisTorres, please visit the https://github.com/natocesarrego/liferay-portal/pull/1255 and apply the same changes requested there

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFileAsStreamCheck.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+							SourceUtil.getIndent(line), newLineStart,
+							StringUtil.merge(
+								parameterList
+							).replace(
+								StringPool.COMMA, StringPool.COMMA_AND_SPACE
+							),
+							");\n\n");
+
+						newContent = StringUtil.replace(
+							newContent, line, newLine + line);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaFileAsStreamCheck.java:84]: @TamyrisTorres, tomorrow morning please reach out me in person for we discuss about this code. I want to understand more about it. Thanks.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Pull request closed without merging**